modules/administration_manual/pages/release_notes.adoc

```diff
@@ -1,5 +1,6 @@
 = Release Notes
 
+* xref:changes-in-10.0.10[Changes in 10.1]
```

Comment:
You need to change `xref:changes-in-10.0.10` to `xref:changes-in-10.1` to match the custom anchor. Alternatively, you could remove the custom anchor and update the xref to be `changes-in-10-1`, as anchors are auto-generated for section titles and headers.

Reply:
Good point :) changed.